extensions/wikia/Scribe/ScribeEventProducer.class.php

```diff
@@ -109,11 +145,14 @@ public function buildEditPackage( $oPage, $oUser, $oRevision = null, $oLocalFile
 		$this->setMediaLinks( $oPage );
 		$this->setTotalWords( str_word_count( $rev_text ) );
 
-		if ( $oLocalFile instanceof File ) {
-			$this->setMediaType( $oLocalFile );
-			$this->setIsImageForReview( ImagesService::isLocalImage( $oTitle ) );
-		} else {
-			$this->setIsImageForReview( false );
+		if ( $oTitle instanceof Title && $oTitle->getNamespace() == NS_FILE ) {
+			$oLocalFile = wfLocalFile( $oTitle );
+			if ( $oLocalFile instanceof File ) {
```

Comment:
According to docs this should be `@return LocalFile|null`, so maybe simple check for is enough. `if ($oLocalFile)`